website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,257 @@
+- node_network_**receive**_packets_total
+- node_network_**transmit**_packets_total  
+ 
+In this example, based on `receive` and `transmit`.
+
+Please follow the same principle and don't put similar metrics details as labels, so the user experience would be fluent.  
+Example for this in an operator:
+- kubevirt_vmi_network_**receive**_errors_total
+- kubevirt_vmi_network_**transmit**_bytes_total
+- kubevirt_migrate_vmi_**data_processed**_bytes
+- kubevirt_migrate_vmi_**data_remaining**_bytes
+
+3. Your metric suffix should indicate the metric unit. For better compatibility, [Prometheus base units](https://prometheus.io/docs/practices/naming/#base-units) should be used.
+4. Prometheus supports four [metric types](https://prometheus.io/docs/concepts/metric_types/#metric-types). `Gauge`,`Counter`,`Histogram` and `Summery`. You can read more about the different types here, [Understanding metrics types](https://prometheus.io/docs/tutorials/understanding_metric_types/#types-of-metrics).  
+The most common types are:
+ - `Counter` - Value can only increase or reset.
+ - `Gauge` Value can be increased and decreased as needed.
+5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
+
+### Prometheus Labels
+[Prometheus labels](https://prometheus.io/docs/practices/naming/#labels) are used to differentiate the characteristics of the thing that is being measured.
+
+1. **Important** - Be cautious when adding labels to metrics. Labels can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values. Note: There are still cases when we will still need to have a high cardinality label like the `pod name`, but try to keep this to the minimum.
+2. When creating a new metric, recording rule or alert, that reports a resource like a `pod` or a `container` name, make sure that the `namespace` is included, in order to be able to uniquely identify it.
+
+#### Metrics `Help` message
+
+Your operator metrics `help` message should include the following details:
+- What does this metric measure?
+- What does the output mean?
+- What important labels does the metric use? (Optional. If applicable).
+
+The `Help` message can be used to create auto-generated documentation, Like it's done here for example in [KubeVirt](https://github.com/kubevirt/kubevirt/blob/main/docs/metrics.md) and generated by the [KubeVirt metrics doc generator](https://github.com/kubevirt/kubevirt/blob/main/tools/doc-generator/doc-generator.go).
+
+We recommend to auto-generated metrics documentation and save it in your operator repository, to a location like to `/docs/monitoring/`, so that the users can find the information about your operator metrics easily. 
```

Comment:
```suggestion
We recommend to auto-generated metrics documentation and save it in your operator repository, to a location like `/docs/monitoring/`, so that the users can find the information about your operator metrics easily. 
```

Reply:
In Memcached operator the metrics.md location defined in Makefile.

Reply:
Fixed